Jenkinsfile

```diff
@@ -1,5 +1,35 @@
 // Define the test routine for different python versions
-def test_python(pythonVersion)
+
+def test_python_basic(pythonVersion)
+{
+  node {
+    // Unstash the source on this node
+    unstash name: 'source'
+    // Set up the environment and test
+    withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: 'clientlibs-test', usernameVariable: 'DB_USER', passwordVariable: 'DB_PASSWORD']]) {
+      try {
```

Comment:
Can we have this `try/catch` in its own method so we don't have as much duplicated code?